feat: cli to show tombstones in transient query output (#6462)
confluentinc/ksql: docs/developer-guide/ksqldb-rest-api/query-endpoint.md

Response JSON Object:

- **header** (object): Information about the result.
    - **header.queryId**: (string): the unique id of the query. This can be useful when debugging. 
    For example, when looking in the logs or processing log for errors or issues.
    - **header.schema**: (string): the list of columns being returned. This defines the schema for 
    the data returned later in **row.columns**.  
- **row** (object): A single row being returned. This will be null if an error is being returned.
    - **row.columns** (array): The values of the columns requested. The schema of the columns was
    already supplied in **header.schema**.
    - **row.tombstone** (boolean): the row is a deletion of a previously row.

>     The **row.key** field contains the unique key of the row that has been deleted.

Author: Good catch.

Response JSON Object:

- **header** (object): Information about the result.
    - **header.queryId**: (string): the unique id of the query. This can be useful when debugging. 
    For example, when looking in the logs or processing log for errors or issues.
    - **header.key**: (string)(since v2): the list of key columns, if the result is a table. 
    This defines the schema for the data returned later in **row.key**. 
    - **header.schema**: (string): the list of columns being returned. This defines the schema for 
    the data returned later in **row.columns**.  
- **row** (object): A single row being returned. This will be null if an error is being returned.

>     - **row.key** (array): If the data being returned is a table, the primary key of the row.

Reviewer: This should be marked as new as of v2.

}
```

### Example stream response

> Where the result of the query is a stream, the response will not include the **row.key** or

Reviewer 2: ```suggestion
If the query result is a stream, the response doesn't include the **row.key** or

```

> **row.tombstone** fields, as streams do not have primary keys.

Reviewer 2: ```suggestion
**row.tombstone** fields, because streams don't have primary keys.

```

```http
HTTP/1.1 200 OK
Content-Type: application/vnd.ksql.v2+json
Transfer-Encoding: chunked

...
{"header":{"queryId":"_confluent_id_19",schema":"`ROWTIME` BIGINT, `NAME` STRING, `AGE` INT"}}
{"row":{"columns":[1524760769983,"1",1524760769747,"alice","home"]}}
...
```

### Example table response

> Where the result of the query is a table, the response will include the primary key of each row in 

Reviewer 2: ```suggestion
If the query result is a table, the response includes the primary key of each row in

```

> the **row.key** field. Deletes from the result table will be identified by the **row.tombstone** 

Reviewer 2: ```suggestion
the **row.key** field. Rows that are deleted from the result table are identified by the **row.tombstone** 

```

field.

```http
HTTP/1.1 200 OK
Content-Type: application/vnd.ksql.v2+json
Transfer-Encoding: chunked

...
{"header":{"queryId":"_confluent_id_34",key":"`ID BIGINT`",schema":"`ROWTIME` BIGINT, `NAME` STRING, `AGE` INT"}}
{"row":{"key":[10],"columns":[1524760769983,"alice",10]}},
{"row":{"key":[10],"tombstone":true}}
...
```

> Note: media type `application/vnd.ksql.v1+json` does not populate **row.key** or return tombstone

Reviewer 2: ```suggestion
!!! note
    Media type `application/vnd.ksql.v1+json` does not populate **row.key** or return tombstone

```
